modular_skyrat/modules/novaya_ert/code/mod_suit.dm

```diff
@@ -54,15 +54,15 @@
 	)
 
 /datum/armor/mod_theme_frontline
```

Comment:
what uses this mod now, the ERT?

Reply:
It's now ERT only, hence the buff; police one's used by the midround spawn.